srcpkgs/residualvm/template

```diff
@@ -20,25 +20,27 @@ build_options="sndio"
 build_options_default="sndio"
 
 if [ -n "$CROSS_BUILD" ]; then
-	configure_args+=" --host=${XBPS_CROSS_TRIPLET}"
+    configure_args+=" --host=${XBPS_CROSS_TRIPLET}"
 
-	pre_configure() {
-		# - sdl-config wrapper already sets --prefix
-		# - freetype-config wrapper already sets --prefix
-		vsed -e "s;_ranlib=ranlib;_ranlib=${XBPS_CROSS_TRIPLET}-ranlib;" \
-		 -e "s;_strip=strip;_strip=${XBPS_CROSS_TRIPLET}-strip;" \
-		 -e "s;_ar=\"ar cru\";_ar=\"${XBPS_CROSS_TRIPLET}-ar cru\";" \
-		 -e "s;_as=\"as\";_as=\"${XBPS_CROSS_TRIPLET}-as\";" \
-		 -e 's;--prefix="$_sdlpath";;' \
-		 -e 's;--prefix="$_freetypepath";;' \
-		 -e "s;^_host_cpu=\"\";_host_cpu=\"${XBPS_TARGET_MACHINE}\";" \
-		 -e 's;^_host_os="";_host_os="linux";' \
-		 -i configure
-	}
+    pre_configure() {
+        # - sdl-config wrapper already sets --prefix
+        # - freetype-config wrapper already sets --prefix
+        vsed -e "s;_ranlib=ranlib;_ranlib=${XBPS_CROSS_TRIPLET}-ranlib;" \
+         -e "s;_strip=strip;_strip=${XBPS_CROSS_TRIPLET}-strip;" \
+         -e "s;_ar=\"ar cru\";_ar=\"${XBPS_CROSS_TRIPLET}-ar cru\";" \
+         -e "s;_as=\"as\";_as=\"${XBPS_CROSS_TRIPLET}-as\";" \
+         -e 's;--prefix="$_sdlpath";;' \
+         -e 's;--prefix="$_freetypepath";;' \
+         -e "s;^_host_cpu=\"\";_host_cpu=\"${XBPS_TARGET_MACHINE}\";" \
+         -e 's;^_host_os="";_host_os="linux";' \
+         -i configure
+    }
 fi
 
 post_install() {
-	# move licenses to the right place
-	vmkdir usr/share/licenses/residualvm
-	mv ${DESTDIR}/usr/share/doc/residualvm/COPYING* ${DESTDIR}/usr/share/licenses/residualvm/
+    # move licenses to the right place
+    vmkdir usr/share/licenses/residualvm
+    for f in COPYING* ; do
+        vlicense ${f}
+    done
```

Comment:
With this change the package contains the `COPYING*` files twice: they’re still installed in `/usr/share/doc/residualvm/`. I think the easiest way is to just move them and ignore the lint error (as it was). All copyright files are installed -- just in the wrong spot. In my opinion it would be acceptable not to use `vlicense` in this special case.

Reply:
Ok fair enough.